f2/src/locales/en.json

```diff
@@ -16,6 +16,7 @@
   "analystReport.dateReceived": "Date received",
   "analystReport.fieldsMissing.warning": "Fields missing from above report",
   "analystReport.flagged": "Flagged",
+  "analystReport.fyiForm": "Flow Type",
```

Comment:
Just wondering about the wording of Flow Type. Will this be clear to the analysts? I think Report Type may be better.

Reply:
@stephaniemgauthier Can we get your opinion here , which one is better " Flow Type" or "Report Type" ?